ena-submission/scripts/get_ena_submission_list.py

```diff
@@ -94,20 +97,31 @@ def filter_for_submission(
             )
             entries_with_external_metadata.add(accession)
         else:
+            # If lower version had external metadata and this one doesn't, remove it from that set
+            entries_with_external_metadata.discard(accession)
+        if entry["metadata"].get("isRevocation", True):
+            logger.info(f"Found revoked sequence: {accession_version}")
```

Comment:
Why do we log this but not all the other cases? Maybe this should just be debug?